src/Xunit.StaFact/Xunit.StaFact.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net472;netstandard2.0;net6.0;net6.0-windows</TargetFrameworks>
+    <TargetFrameworks>net472;netstandard2.0;net6.0;net6.0-windows;net6.0-macos10.15</TargetFrameworks>
+    <SupportedOSPlatformVersion Condition="'$(TargetingMacos)'=='true'">10.15</SupportedOSPlatformVersion>
```

Comment:
I'm surprised this property isn't set by virtue of the TargetFramework mentioning the macos version. Are you sure this is necessary?

Reply:
In theory, setting this is supposed to allow us to target `net6.0-macos` instead of `net6.0-macos10.15`, and still get `net6.0-macos10.15` in the nupkg, but it wasn't working and we need to investigate. If we don't specify 10.15 _somewhere_, it uses the OS version of whoever does the build.

But yeah, it's not needed if we explicitly target `net6.0-macos10.15`.

Reply:
> If we don't specify 10.15 somewhere, it uses the OS version of whoever does the build.

Are you sure? I can build `net6.0-macos` from Windows, and that has no basis for a macos version.
Also, when I target `net6.0-windows`, that targets Windows 7 -- not the version of Windows that I am using.

Reply:
It looks like each TFW can specify its own default OS version behavior: https://learn.microsoft.com/en-us/dotnet/standard/frameworks#os-version-in-tfms. So I would imagine the macOS workload targets are doing this.

And I think I see my answer for `SupportedOSPlatformVersion` here: https://learn.microsoft.com/en-us/dotnet/standard/frameworks#support-older-os-versions. Seems that `SupportedOSPlatformVersion` indicates runtime version. So I should drop it anyway.